package.json

```
@@ -47,7 +47,7 @@
     "@typescript-eslint/utils": "7.0.1",
     "arg": "5.0.2",
     "batching-toposort": "1.2.0",
-    "buffer": "6.0.3",
+    "buffer": "^6.0.3",
```

```suggestion
    "buffer": "6.0.3",
```
Keep the pin, we can let renovate handle the updates